docs/components/arm/_index.md

```diff
@@ -26,6 +26,14 @@ When controlling an arm with `viam-server`, the following features are implement
 - Self-collision prevention
 - Obstacle avoidance
 
+## Related Services
```

Comment:
I think it might be good putting this above supported models here as well just for consistency